pom.xml

```diff
@@ -60,7 +60,7 @@
         <dep.testng.version>7.5</dep.testng.version>
         <dep.lucene.version>8.10.0</dep.lucene.version>
         <dep.assertj-core.version>3.8.0</dep.assertj-core.version>
-        <dep.logback.version>1.2.13</dep.logback.version>
+        <dep.logback.version>1.5.13</dep.logback.version>
```

Comment:
Instead of adding all these exclusions, 
1. remove this dep.logback.version declaration
2. upgrade logback version in airbase (the parent pom for this and the various dependencies you've added exclusions for)
3. release new version of airlift
4. Upgrade the airbase version used in all these dependencies (airlift, benchto).
5. Release new versions of any libraries we upgraded
6. update the airbase version in presto and update the versions for all the libraries we did new releases of.